client/src/notice.ts

```diff
@@ -1,28 +1,26 @@
 import { Intent, Position, Toaster } from '@blueprintjs/core';
 
-import { defaultNoticeTimeout } from 'corla/config';
-
 
 const Toast = Toaster.create({ autoFocus: true });
 
 
-function danger(message: string, timeout: number = defaultNoticeTimeout) {
+function danger(message: string, timeout: number = 0) {
     Toast.show({
         intent: Intent.DANGER,
         message,
         timeout,
     });
 }
 
-function warning(message: string, timeout: number = defaultNoticeTimeout) {
+function warning(message: string, timeout: number = 0) {
     Toast.show({
         intent: Intent.WARNING,
         message,
         timeout,
     });
 }
 
-function ok(message: string, timeout: number = defaultNoticeTimeout) {
+function ok(message: string, timeout: number = 10000) {
```

Comment:
Why is this now not a symbolic constant anymore?

Reply:
Because there are now 3 degrees of freedom, instead 1 default. I'll go ahead and add two constants back to the config. It is nice to have it in one place.